src/mmw/js/src/analyze/views.js

```diff
@@ -60,9 +61,13 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
     },
 
     showHeaderRegion: function() {
-        this.headerRegion.show(new HeaderView({
-            model: new models.AnalyzeModel({
-                shape: this.model.get('area_of_interest')
+        this.headerRegion.show(new coreViews.AreaOfInterestView({
+            App: App,
+            model: new coreModels.AreaOfInterestModel({
+                shape: this.model.get('area_of_interest'),
+                canGoBack: true,
```

Comment:
I think we could use snake_case for all model attributes. They are almost never accessed directly in JavaScript, unlike properties and methods like `this.headerRegion.show`, and almost always accessed as string arguments to Backbone.Model.get. Thus using snake_case for all model attributes would be predictable without breaking JavaScript convention.